api/serverApiFunctions.ts

```diff
@@ -0,0 +1,12 @@
+'use server';
+import { users } from './serverConfig';
+
+export const deleteUser = async (formData: FormData) => {
+  const userId = formData.get('userId') as string;
+  try {
+    await users.delete(userId);
+    return { success: true };
+  } catch (error) {
+    throw new Error('Error Deleting User');
+  }
+};
```

Comment:
Will we move all of API calls from the current apiFunctions.ts file to this one?

Reply:
Nope. They will remain in apiFunctions as they use the client SDK.